dashboard/app/views/home/_homepage.html.haml

```diff
@@ -34,10 +34,24 @@
         Manage Applications
   -if can? :read, Pd::Workshop
     %h1
-      Workshop Dashboard
+      Workshop Resources
```

Comment:
If user can manage applications (e.g. admins for regional partners) they currently see this
<img width="396" alt="Screen Shot 2020-01-30 at 2 35 58 PM" src="https://user-images.githubusercontent.com/46507039/73496622-dc047f80-436d-11ea-9495-25175a2647c0.png">
Should we use consistent language for both application and workshop headers?

Reply:
Oh good point. Instead I'll just make a new header. One sec